rules/rules-reviewed/jakarta-ee9/java-ee/spring-components.windup.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<ruleset id="spring-components" xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset ensures the versions of Spring and Spring Boot are Jakarta EE 9 compliant
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="jakarta-ee" versionRange="[9,)" />
+        <targetTechnology id="jws" versionRange="[6,)" />
+    </metadata>
+    <rules>
+        <rule id="spring-components-00001">
+            <!-- Spring Boot version 3.0.0 is Jakarta compliant -->
+            <when>
+                <or>
+                    <dependency groupId="org.springframework.boot" artifactId="{*}" toVersion="2.9.9" />
```

Comment:
I can see the latest SB version is `2.7.7` but we could be more defensive in the `toVersion` maybe having something like `2.99.999` to be deliberately high with the version to ensure we will be able to cover many further SB 2 maintenance release

Reply:
more defensive versioning applied